_posts/2020-08-26-bazel-3.5.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: posts
+title: "Bazel 3.5"
+authors:
+  - aiuto
+---
+
+[Bazel 3.5](https://github.com/bazelbuild/bazel/releases/tag/3.5.0) has just been released.
+
+Bazel 3.5 is a minor release. It is an incremental update to Bazel 3.4.
+
+Basline: 889bc0b523b47eeb38a72bf9bb6858ee525a7c7e
+
+Cherry picks:
```

Comment:
Remove the list of cherrypicks, a blog post should be human-readable text.
(the exact information of what goes in the binary is on GitHub)

Reply:
> Nothing about bazelcon?

That's not really relevant for release notes.